environment.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
   - pytest-mock
   - sphinx
   - numpydoc
+  - typing_extensions
```

Comment:
We might need to require this if it causes import failures with 3.5.1

Reply:
I am setting up the environment and C compiler on my laptop. I will run tests and let you know if it fails.

Reply:
@WillAyd does it matter how I create virtualenv(anaconda, pipenv) when running tests locally. My system is low on space and anaconda takes up a lot of space. So is it ok if I use pipenv to create env and then run `pytest pandas` or will that make any difference.